resources/views/layouts/default/profile/index.blade.php

```diff
@@ -21,8 +21,8 @@
       </p>
       <h6><span class="flag-icon flag-icon-{{ $user->country }}"></span></h6>
       <div class="social-description">
+        <img src="{{ $user->rank->image_url }}" style="width: 160px;">
```

Comment:
I think it would make sense to wrap this:

```
@if (!empty($user->rank->image_url))
 <img ...>
@endif
```

Reason being that if an image_url isn't defined, I think some browsers might show a broken image thing. Might need to be checked, though

Reply:
I have added this suggestion into the code, and pushed